index.js

```diff
@@ -10,9 +10,10 @@ module.exports = function(str) {
     return '';
   }
 
+  var index = 0;
   return str.split(newlineRegex).map(function(line) {
     if (line.match(newlineRegex)) {
-      return React.createElement('br');
+      return React.createElement('br', { key: index++ });
```

Comment:
The second parameter of the callback we provide for mapping already contains the index :
``` js
return str.split(newlineRegex).map(function(line, index) {
  if (line.match(newlineRegex)) {
    return React.createElement('br', {key: index});
  } else {
    return line;
  }
```